website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -3749,6 +3749,11 @@ the user ids from a `PCollection` of purchases one would write (using the `Selec
 purchases.apply(Select.fieldNames("userId"));
 {{< /highlight >}}
 
+{{< highlight py >}}
+input_pc = ... # {"user_id":...,"bank": ..., "purchase_amount": ...}
```

Comment:
```suggestion
input_pc = ... # {"user_id": ...,"bank": ..., "purchase_amount": ...}
```

Reply:
I was actually thinking about the same, if we could represent it in a kind off pretty json format it would provide better understandability of the input